app/views/components/fields/field_accessible_autocomplete_with_header.scala.html

```diff
@@ -47,7 +47,7 @@
 
     @if(isPageHeading) {<h1 class="govuk-label-wrapper">}
     <label id="@(fieldNameWithoutBrackets)-label" for="@fieldNameWithoutBrackets" class="govuk-label@if(labelClass.nonEmpty){ @labelClass}" >
-        @messages(label)
+        @if(label.nonEmpty) { @messages(label) }
```

Comment:
This change was added to fix country_of_routing change which is (I believe) incorrectly using an empty <label>